src/main/java/uk/gov/hmcts/reform/demo/controllers/HelloWorldEndpoint.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.demo.controllers;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import static org.springframework.http.ResponseEntity.ok;
+
+/**
+ * Default endpoints per application.
+ */
+@RestController
+public class HelloWorldEndpoint {
```

Comment:
Is this controller supposed to be renamed by the users of this template? if not, let's use a less silly name.